data/human/kestrel.txt

```diff
@@ -329,32 +327,42 @@ ship "Kestrel"
 
 
 ship "Kestrel" "Kestrel (More Engines)"
+	sprite "ship/kestrelE"
```

Comment:
I'd follow the Marauder image naming scheme where the suffix isn't capitalized. That is, make the following changes:
* kestrelE to kestrele
* kestrelS to kestrels
* kestrelW to kestrelw
* kestrelO to kestrelo

Reply:
Fixed. Changed kestrelo to kestreloriginal for more clarification also.